daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala

```diff
@@ -169,6 +169,10 @@ object TransactionCoder {
                 .addAllSignatories(e.signatories.map(_.underlyingString).asJava)
                 .addAllStakeholders(e.stakeholders.map(_.underlyingString).asJava)
 
+            if (transactionVersion precedes minNoControllers) {
+              nodeBuilder.setExercise(exBuilder).build()
```

Comment:
Did you intend to clear the controllers from `exBuilder` instead of this noop statement?

Reply:
done